src/CoolPropLib.cpp

```diff
@@ -101,99 +101,101 @@ double convert_from_SI_to_kSI(long iInput, double value)
 }
 
 EXPORT_CODE long CONVENTION redirect_stdout(const char* file){
+    fpu_reset_guard guard;
     FILE *fp = freopen(file, "a+", stdout);
-    reset_fpu();
```

Comment:
Actually, I don't know why fpu resetting is required here. Does freopen() do floating-point math?